webroot/js/app/components/staggered-list.js

```diff
@@ -12,8 +12,8 @@ Vue.component('staggered-list', {
     template: `
         <transition-group appear
             name="${NAME}"
-            v-on:enter=enter
-            >
+            v-on:enter="enter"
+            v-on:after-enter=afterEnter>
```

Comment:
Just to be coherent with previously used syntax: `v-on:after-enter="afterEnter"`